proposals/i18n-getOSLanguage.md

```diff
@@ -0,0 +1,74 @@
+# Proposal: i18n.getOSLanguage>
+
+**Summary**
+
+Allow to get the language of your operating system as an `IETF tag`
+
+**Document Metadata**
+
+**Author:** carlosjeurissen
+
+**Created:** 2024-03-18
+
+**Related Issues:** https://github.com/w3c/webextensions/issues/252
+
+## Motivation
+
+### Objective
+
+As browsers often do not directly use the OS language but some reflection of it based on what languages are supported by the browser UI.
```

Comment:
This reads a little incompletely to me.

Maybe:

```suggestion
Allow extensions to display UI in the user's preferred language, even when this is distinct from the browser's UI language (which is restricted to languages supported by the browser).
```

Feel free to wordsmith.

Reply:
Rephrased to:
`Allow extensions to display UI in the user's system language, even when this is distinct from the browser's UI language (which is restricted to languages supported by the browser).`